src/main/java/seedu/address/logic/commands/SummariseCommand.java

```diff
@@ -165,7 +165,7 @@ private String summariseBlock(List<Person> result, String blockLetter) {
      * @param list the unfiltered entire list in the database
```

Comment:
Can correct this the sentence a bit weird

Reply:
Hi! Thank you for pointing that out. I guess it was hard to understand. I have simplified the description of this param found in the rest of the methods too. Thank you. :stuck_out_tongue_closed_eyes:

Reply:
Fantastic! Thank you jiale 😄